scripts/install

```diff
@@ -33,6 +32,7 @@ instance_name=$YNH_APP_ARG_NAME
 registration=$YNH_APP_ARG_REGISTRATION
 admin_email=$(ynh_user_get_info $admin 'mail')
 secret_key=$(ynh_string_random --length=32 | base64)
+architecture=$YNH_ARCH
```

Comment:
From where `$YNH_ARCH` is initialised ?

Reply:
`$YNH_ARCH` replaces `ynh_detect_arch` helper
https://github.com/YunoHost/yunohost/blob/fa163cbc2e44a5c54ac7d48adff7c573da6c2b30/debian/changelog#L93

https://github.com/YunoHost/yunohost/blob/fa163cbc2e44a5c54ac7d48adff7c573da6c2b30/src/yunohost/app.py#L2393

Reply:
Ok cool, would be great to have some information about that in the documentation ...